### ISSUE TITLE
Feature request: Next Generation Methods

### ISSUE THREAD

**Issue by timothyslau** [SciML/ReservoirComputing.jl#108]
I found this new reservoir computing method after reading the paper below
https://github.com/quantinfo/ng-rc-paper-code

"Next generation reservoir computing | Nature Communications" https://www.nature.com/articles/s41467-021-25801-2

Is anyone able to convert the python code to Julia?

**Comment by MartinuzziFrancesco** [comment]
Yes, that is definitely a high priority implementation for ReservoirComputing.jl. There will be a slight refactorization of the code base in the following weeks, and after that I'll tackle NG-RC
